### PR TITLE
create symbology from statistics

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Colorizer für neuen Layer dynamisch erzeugen](https://trello.com/c/ggWittnR/1108-colorizer-f%C3%BCr-neuen-layer-dynamisch-erzeugen)

File: projects/core/src/lib/layers/symbology/symbology-creator/symbology-creator.component.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <mat-form-field appearance="fill">
+        <mat-label>Symbology</mat-label>
+        <mat-select [formControl]="value">
+            <mat-option [value]="AS_INPUT">Same as input</mat-option>
+            <mat-option [value]="COMPUTE_LINEAR_GRADIENT">Compute linear gradient</mat-option>
+        </mat-select>
+    </mat-form-field>
+    <div class="description">
+        <mat-hint [ngSwitch]="value.value">
+            <span *ngSwitchCase="AS_INPUT" class="mat-caption">Use the smbology of the input for the new layer.</span>
```
Comment (jdroenner) [inline projects/core/src/lib/layers/symbology/symbology-creator/symbology-creator.component.html:11]: symbology

**Review by jdroenner — CHANGES_REQUESTED** [review]

File: projects/core/src/lib/layers/symbology/symbology-creator/symbology-creator.component.html (hunk 1 above)
Comment (ChristianBeilschmidt) [reply projects/core/src/lib/layers/symbology/symbology-creator/symbology-creator.component.html]: [e2f95ba](https://github.com/geo-engine/geoengine-ui/pull/524/commits/e2f95baf61c99dd25e4ba43dae805cbec175a966)

Was fixed among other things… here :wink:

**Review by jdroenner — APPROVED** [review]

**Comment by jdroenner** [comment]
bors merge

**Pull request merged**